db/schema.rb

```diff
@@ -110,6 +110,7 @@
     t.integer "max_members"
     t.boolean "students_are_repo_admins", default: false, null: false
     t.boolean "invitations_enabled", default: true
+    t.integer "max_teams"
```

Comment:
I thought this already merged? You might have some other commits in this PR.

Reply:
Oops I will fix that.

Reply:
It looks like this reappeared again